WalletWasabi.Tests/UnitTests/WabiSabi/Client/BobClientTests.cs

```diff
@@ -53,7 +51,7 @@ public async Task RegisterOutputTestAsync()
 			Assert.Equal(Phase.InputRegistration, round.Phase);
 
 			var bitcoinSecret = km.GetSecrets("", coin1.ScriptPubKey).Single().PrivateKey.GetBitcoinSecret(Network.Main);
-			var aliceClient = await AliceClient.CreateNewAsync(aliceArenaClient, new[] { coin1.Coin }, bitcoinSecret, round.Id, round.Hash, round.FeeRate);
+			var aliceClient = await AliceClient.CreateNewAsync(aliceArenaClient, coin1.Coin, bitcoinSecret, round.Id, round.Hash, round.FeeRate);
```

Comment:
aside: i don't really understand why `coin1` is a `SmartCoin` - is it to set up the mock parameters (height & txout?)

Reply:
Because `BitcoinFactory` creates `SmartCoin`s

Reply:
using `BitcoinFactory` to create coins is different from most of the other wabisabi tests, where `WabiSabiFactory` creates `Alice`s complete with coins ready for use